app/src/main/java/com/example/hw4/fragments/DetailsFragment.kt

```diff
@@ -0,0 +1,54 @@
+package com.example.hw4.fragments
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.os.bundleOf
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.fragment.navArgs
+import com.example.hw4.R
+import com.example.hw4.databinding.FragmentAddNoteBinding
+import com.example.hw4.databinding.FragmentDetailsBinding
+
+class DetailsFragment : Fragment() {
+
+    private var _binding: FragmentDetailsBinding? = null
+    private val binding get() = requireNotNull(_binding)
+
+    private val args by navArgs<DetailsFragmentArgs>()
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return FragmentDetailsBinding.inflate(inflater, container, false)
+            .also { _binding = it }
+            .root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        with(binding) {
+            titleDetails.text = args.title
+            descriptionDetails.text = args.description
+            dateDetails.text = args.date
+
+            val dialog = DialogChangeFragment()
+            val bundle = Bundle()
+            bundle.putLong("idKey", args.id)
+            dialog.arguments = bundle
```

Comment:
либо так, либо через метод getInstance в DialogFragment
```suggestion
            val dialog = DialogChangeFragment().apply {
                arguments = bundleOf("idKey", args.id)
            }
```